packages/governance/src/contractHelper.js

```diff
@@ -80,20 +80,24 @@ const handleParamGovernance = (zcf, paramManager) => {
   };
 
   /**
-   * @template CF
-   * @param {CF} originalCreatorFacet
-   * @returns { GovernedCreatorFacet<CF> }
+   * @template {GovernableCreatorFacet} CF creator facet
+   * @param {ERef<CF>} originalCreatorFacet
+   * @param {Record<string, (...args: any[]) => any>} governedApis
+   * @returns { ERef<GovernedCreatorFacet<CF>> }
    */
-  const wrapCreatorFacet = originalCreatorFacet => {
+  const wrapCreatorFacet = (originalCreatorFacet, governedApis = {}) => {
     const limitedCreatorFacet = makeLimitedCreatorFacet(originalCreatorFacet);
 
     // exclusively for contractGovernor, which only reveals limitedCreatorFacet
     return Far('creatorFacet', {
+      // @ts-expect-error special case
       getParamMgrRetriever: () => {
         return Far('paramRetriever', { get: () => paramManager });
       },
       getInvitation: name => paramManager.getInternalParamValue(name),
       getLimitedCreatorFacet: () => limitedCreatorFacet,
+      getGovernedApis: () => Far('governedAPIs', governedApis),
+      getGovernedApiNames: () => Object.keys(governedApis),
```

Comment:
We talked about getting the API names from the governed API remote call. IIUC we can't because the whole object is Far and there's no way to make it an copyable object of method names mapped to Far methods. Is that right?

I request that be documented as explanation for the surprise that the API requires a `getGovernedApiNames` that is seemingly redundant with getting the keys of what's returned by `getGovernedApis`.